app/controllers/application_controller.rb

```diff
@@ -1,10 +1,149 @@
 require './config/environment'
+require 'pry'
 
 class ApplicationController < Sinatra::Base
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
   end
 
```

Comment:
we should add some helper methods here like `logged_in?` and `current_user` that will allow us to abstract repeat patterns that we can use in our controllers and view. 

```ruby 
def logged_in?
  !!current_user 
end 

def current_user 
  @current_user ||= User.find_by_id(session[:id])
end
```